### DISCUSSION TITLE
assetManager.open function is not allow me to upload the image

### DISCUSSION THREAD

**Discussion by pardeepsinghrajewal** [GrapesJS/grapesjs#5580] [Q&A]
I'm a newcomer to GrapesJS, and I've recently set up a new plugin using the GrapesJS CLI available at https://github.com/GrapesJS/cli.

Currently, when attempting to open the assetManager with assetManager.open(), I encounter an interface displaying the upload option, as shown here: https://nimb.ws/xIoWU6

**Comment by artf** [comment]
please share the code
